app/src/behaviors/behavior_tap_dance.c

```diff
@@ -258,20 +266,28 @@ static int behavior_tap_dance_init(const struct device *dev) {
 #define _TRANSFORM_ENTRY(idx, node) ZMK_KEYMAP_EXTRACT_BINDING(idx, node)
 
 #define TRANSFORMED_BINDINGS(node)                                                                 \
-    {LISTIFY(DT_INST_PROP_LEN(node, bindings), _TRANSFORM_ENTRY, (, ), DT_DRV_INST(node))}
+    {LISTIFY(DT_PROP_LEN(node, bindings), _TRANSFORM_ENTRY, (, ), node)}
+
+#define BREAK_ITEM(i, inst) DT_PROP_BY_IDX(inst, binding_params, i)
+#define BREAK_ZERO(i, inst) 0
 
-#define KP_INST(n)                                                                                 \
+#define KP_INST(inst)                                                                              \
```

Comment:
Might be a good opportunity to rename this to e.g. `TD_INST`, similar to other recently updated behaviors.